### PR TITLE
Squishy303

### PR DESCRIPTION
**Pull request by LegoStormtroopr**
Issues resolved: #

Sanity checks:
- [ ] Have tests been run locally?
- [ ] Does this change any javascript? If so, have you checked that the UI works as expected?
- [ ] If there are data model changes, are relevant data standards referenced - in code?
- [ ] If there are deviations from any specified standards, are the reasons for changes documented?
- [ ] If there are UI changes have you documented whats changed so the help documentation can be updated?

### CODE REVIEW

File: python/aristotle-dataset-extensions/aristotle_dse/migrations_old/0001_squashed_0013_now_with_dcat.py [hunk 1]
```diff
@@ -11,7 +11,7 @@ class Migration(migrations.Migration):
     replaces = [('aristotle_dse', '0001_initial'), ('aristotle_dse', '0002_auto_20150510_0249'), ('aristotle_dse', '0003_add_clusters_improve_names'), ('aristotle_dse', '0004_auto_20160103_0200'), ('aristotle_dse', '0005_improve_inculsion_help_text'), ('aristotle_dse', '0006_auto_20160103_1855'), ('aristotle_dse', '0007_add_ordering_to_inclusions'), ('aristotle_dse', '0008_auto_20160318_2335'), ('aristotle_dse', '0009_auto_20160725_1958'), ('aristotle_dse', '0010_datasource_specification'), ('aristotle_dse', '0011_auto_20160726_2015'), ('aristotle_dse', '0012_fix_concept_fields'), ('aristotle_dse', '0013_now_with_dcat')]
 
     dependencies = [
-        ('aristotle_mdr_identifiers', '0001_initial'),
+        # ('aristotle_mdr_identifiers', '0001_initial'),
```
Comment (harrywhite4) [inline python/aristotle-dataset-extensions/aristotle_dse/migrations_old/0001_squashed_0013_now_with_dcat.py]:
```suggestion
        ('aristotle_mdr_identifiers', '0001_initial'),
```

File: python/aristotle-dataset-extensions/aristotle_dse/migrations_old/0022_auto_20190501_1043.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+from django.db import migrations
+from aristotle_mdr.utils.migrations import CustomFieldMover
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('aristotle_mdr', '0057_auto_20190329_1609'),
+        ('aristotle_dse', '0021_auto_20190415_0012'),
+        # ('aristotle_mdr_custom_fields', '0002_auto_20190212_1805'),
```
Comment (harrywhite4) [inline python/aristotle-dataset-extensions/aristotle_dse/migrations_old/0022_auto_20190501_1043.py]:
```suggestion
        ('aristotle_mdr_custom_fields', '0002_auto_20190212_1805'),
```

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/reviews/migrations/0001_initial.py [hunk 3]
```diff
@@ -14,7 +14,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('aristotle_mdr', '0045__concept_superseded_by_items'),
+        # ('aristotle_mdr', '0045__concept_superseded_by_items'),
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/reviews/migrations/0001_initial.py]: Not sure if this was intended to be removed

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/migrations/0001_squashed_0008_initial.py [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+import django.utils.timezone
+import model_utils.fields
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        # ('aristotle_mdr', '0075_auto_20190916_1602'),
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/migrations/0001_squashed_0008_initial.py]: Should be dependant on aristotle_mdr

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0024_synonym_data_migration.py [hunk 5]
```diff
@@ -40,9 +40,9 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('aristotle_mdr', '0023_auto_20180206_0332'),
-        ('aristotle_mdr_slots', '0004_switch_to_concept_relations')
+        # ('aristotle_mdr_slots', '0004_switch_to_concept_relations')
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0024_synonym_data_migration.py]:
```suggestion
        ('aristotle_mdr_slots', '0004_switch_to_concept_relations')
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0024_synonym_data_migration.py [hunk 6]
```diff
@@ -40,9 +40,9 @@ class Migration(migrations.Migration):
...
     ]
 
     operations = [
-        migrations.RunPython(add_slots, reverse_add_slots),
+        # migrations.RunPython(add_slots, reverse_add_slots),
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0024_synonym_data_migration.py]:
```suggestion
        migrations.RunPython(add_slots, reverse_add_slots),
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0029_remove__concept_short_name.py [hunk 7]
```diff
@@ -19,11 +19,11 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('aristotle_mdr', '0028_replace_old_ded_through'),
-        ('aristotle_mdr_slots', '0006_auto_20180531_0337')
+        # ('aristotle_mdr_slots', '0006_auto_20180531_0337')
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0029_remove__concept_short_name.py]:
```suggestion
        ('aristotle_mdr_slots', '0006_auto_20180531_0337')
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0029_remove__concept_short_name.py [hunk 8]
```diff
@@ -19,11 +19,11 @@ class Migration(migrations.Migration):
...
     ]
 
     operations = [
-        migrations.RunPython(add_slots, reverse_add_slots),
+        # migrations.RunPython(add_slots, reverse_add_slots),
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0029_remove__concept_short_name.py]:
```suggestion
        migrations.RunPython(add_slots, reverse_add_slots),
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0041_migrate_favourites.py [hunk 9]
```diff
@@ -33,9 +33,9 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('aristotle_mdr', '0040_rename_favourites'),
-        ('aristotle_mdr_favourites', '0004_auto_20180913_2021')
+        # ('aristotle_mdr_favourites', '0004_auto_20180913_2021')
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0041_migrate_favourites.py]:
```suggestion
        ('aristotle_mdr_favourites', '0004_auto_20180913_2021')
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0041_migrate_favourites.py [hunk 10]
```diff
@@ -33,9 +33,9 @@ class Migration(migrations.Migration):
...
     ]
 
     operations = [
-        migrations.RunPython(migrate_favs_forward, migrate_favs_backward)
+        # migrations.RunPython(migrate_favs_forward, migrate_favs_backward)
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0041_migrate_favourites.py]:
```suggestion
        migrations.RunPython(migrate_favs_forward, migrate_favs_backward)
```

File: python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0058_auto_20190502_0853.py [hunk 11]
```diff
@@ -10,7 +10,7 @@ class Migration(migrations.Migration):
 
     dependencies = [
         ('aristotle_mdr', '0057_auto_20190329_1609'),
-        ('aristotle_mdr_custom_fields', '0002_auto_20190212_1805'),
+        # ('aristotle_mdr_custom_fields', '0002_auto_20190212_1805'),
```
Comment (harrywhite4) [inline python/aristotle-metadata-registry/aristotle_mdr/migrations_old/0058_auto_20190502_0853.py]:
```suggestion
        ('aristotle_mdr_custom_fields', '0002_auto_20190212_1805'),
```

File: python/aristotle-metadata-registry/aristotle_mdr/views/wizards.py [hunk 12]
```diff
@@ -703,7 +703,7 @@ class DataElementWizard(MultiStepAristotleWizard):
         ("find_de_from_comp", MDRForms.wizards.DE_Find_DE_Results_from_components),
         ("find_dec_results", MDRForms.wizards.DE_Find_DEC_Results),
         ("make_dec", MDRForms.wizards.subclassed_wizard_2_Results(MDR.DataElementConcept)),
-        ("make_vd", MDRForms.wizards.subclassed_wizard_2_Results(MDR.ValueDomain)),
+        ("make_vd", MDRForms.wizards.subclassed_wizard_2_Results(MDR.DataElementConcept)),
```
Comment (s-i-l-k-e) [inline python/aristotle-metadata-registry/aristotle_mdr/views/wizards.py]:
```suggestion
        ("make_vd", MDRForms.wizards.subclassed_wizard_2_Results(MDR.ValueDomain)),
```

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/reviews/migrations/0001_initial.py (hunk 3 above)
Comment (LegoStormtroopr) [reply python/aristotle-metadata-registry/aristotle_mdr/contrib/reviews/migrations/0001_initial.py]: It wasn't

File: python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/migrations/0001_squashed_0008_initial.py (hunk 4 above)
Comment (LegoStormtroopr) [reply python/aristotle-metadata-registry/aristotle_mdr/contrib/slots/migrations/0001_squashed_0008_initial.py]: Yup.

**Pull request merged**